superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -955,6 +955,18 @@ export const controls = {
     description: 'Whether to include a time filter',
   },
 
+  filter_order: {
+    type: 'SelectControl',
+    multi: true,
```

Comment:
but don't you run out of options then? each option can only be used once isn't it?